### ISSUE TITLE
Create end-to-end test for platform brightness on Android

### ISSUE THREAD

**Issue by matthew-carroll** [flutter/flutter#42990]
Create end-to-end test for platform brightness on Android.

The Android embedding is responsible for reporting Android's platform brightness to Flutter, either light or dark. This ticket is to setup an end-to-end test that verifies that real platform brightness values are correctly reported from the Android embedding to Flutter over the settings channel.

**Comment by Hixie** [comment]
@matthew-carroll have we made any progress in making this testable?

**Comment by matthew-carroll** [comment]
@Hixie I don't believe anyone is working on, nor am I aware of any team that has scheduled this work on their roadmap. I think the add-to-app team is already pretty busy for the quarter. This could be a good infra project, but I'm not sure what their focus is at the moment.

It's also possible that this exploration would make sense for the framework, too, because the boundary between the framework and the platform appears to create an issue for testing other behaviors, too, such as keyboard input.

**Comment by Hixie** [comment]
Can we build a bespoke devicelab test that just runs the app and then artificially switches the mode by toggling the setting in the quick settings bar or some such?
